packages/core/admin/admin/src/core/apis/Library.js

```diff
@@ -0,0 +1,13 @@
+// @HichamELBSI, @mfrachet if you have a better naming for our components and fields
+// store I will be happy to change the current name
+import Components from './Components';
+import Fields from './Fields';
+
+class Library {
```

Comment:
The `Library` class is only for fields and components?
What is the benefit of having this intermediate class?

Reply:
You are right it can be an object.